pennylane/templates/embeddings/angle.py

```diff
@@ -66,26 +40,41 @@ def AngleEmbedding(features, wires, rotation="X"):
     ``features`` than rotations, the circuit does not apply the remaining rotation gates.
 
     Args:
-        features (array): input array of shape ``(N,)``, where N is the number of input features to embed,
+        features (tensor-like): input tensor of shape ``(N,)``, where N is the number of input features to embed,
```

Comment:
tensor-like is more accurate than "array"

Reply:
(I think we use `tensor_like` elsewhere in the docs, not sure if sphinx parses a hyphen correctly!)